roles/openshift_common/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_facts
-- role: openshift_repos
+- role: openshift_docker
```

Comment:
Probably not, per other comment we can likely just query in version, and leave this guy as he was.